ocaml/quicktest/storage_test.ml

```diff
@@ -206,6 +206,10 @@ module Sr_filter = struct
   let not_type _type rpc session_id =
     List.filter (fun i -> Client.Client.SR.get_type ~rpc ~session_id ~self:i.sr <> _type)
 
+  let thin_pro rpc session_id =
+    List.filter (fun i -> match Client.Client.SR.get_type ~rpc ~session_id ~self:i.sr with
+        | "gfs2" | "lvm" -> true | _ -> false)
```

Comment:
I'm confused, LVM is not thinly provisioned, if it was we wouldn't need GFS2.
What is the actual property that you need in your test? Is it enough free space to accomodate the 2GB VDI?

Reply:
Good point, *I* was confused. What I want for this test is a sparse VDI, so that we can quickly copy only the allocated blocks. This is true for lvm, because, even though we allocate all the space, there is a VHD inside it. (And the filer can still make it thin-pro, if it's intelligent, it can lie and say the space is allocated even though it isn't) In XS we currently have 2 types of VDIs, raw and sparse, so what we want for this test is a sparse VDI, not a raw one. I think all the SRs support sparse VDIs (maybe except for the ISO SR).